src/Tribe/Admin/Manager/Service_Provider.php

```diff
@@ -117,4 +120,71 @@ public function render_error( $error_message ) {
 
 		return $error_message;
 	}
+
+	/**
+	 * Add the Attendee Report nav button to WP Admin Nav bar.
+	 *
+	 * @since TBD
+	 */
+	public function add_attendees_view_button() {
+
+		// Check user permission.
+		if ( ! current_user_can( 'edit_posts' ) ) {
+			return;
+		}
+
+		$url     = '';
+		$post_id = 0;
+
+		// Set the URL for WP Admin.
+		if ( is_admin() && 'edit' == tribe_get_request_var( 'action' ) ) {
+			$post_id = tribe_get_request_var( 'post' );
+		}
+
+		// Set the URL for Front-end.
+		if ( is_single() ) {
+			$post = get_post();
+
+			if ( empty( $post ) ) {
+				return;
+			}
+
+			$post_id = $post->ID;
+		}
+
+		// If no valid post is found, bail out.
+		if ( 0 == $post_id ) {
```

Comment:
Let's use a strict comparison here
```suggestion
		if ( 0 === $post_id ) {
```